WcaOnRails/app/views/search_results/index.html.erb

```diff
@@ -77,6 +77,21 @@
       <%= paginate @posts, param_name: :posts_page, params: { anchor: "posts" } %>
     <% end %>
 
+    <% if !@regulations.empty? %>
+      <h2><%= anchorable "Regulations and Guidelines" %></h2>
+      <% @regulations.each do |reg| %>
+        <div>
+          <%= link_to reg["url"] do %>
+            <h4><%= wca_highlight(reg["id"], query_parts) %></h4>
+          <% end %>
+          <blockquote>
+            <%= wca_highlight(reg["text"], query_parts) %>
```

Comment:
Ah, darn. Perhaps it's our calls to `strip_tags` here: https://github.com/cubing/worldcubeassociation.org/blob/9881b67d34044801b8c72b6cd622a0bfd74162dd/WcaOnRails/app/helpers/application_helper.rb#L72

Does this work:

```
highlight(reg["text"].html_safe, query_parts)
```